test/test_inverse.py

```diff
@@ -177,10 +181,170 @@ def test_KFAC_inverse_damped_matmat(
         aaT.sub_(torch.eye(aaT.shape[0], device=aaT.device), alpha=delta)
     for ggT in KFAC._gradient_covariances.values():
         ggT.sub_(torch.eye(ggT.shape[0], device=ggT.device), alpha=delta)
-    inv_KFAC = KFACInverseLinearOperator(KFAC, damping=(delta, delta), cache=cache)
+    # as a single scalar
+    inv_KFAC = KFACInverseLinearOperator(KFAC, damping=delta, cache=cache)
+    # and as a tuple
+    inv_KFAC_tuple = KFACInverseLinearOperator(
+        KFAC, damping=(delta, delta), cache=cache
+    )
+
+    num_vectors = 2
+    X = random.rand(KFAC.shape[1], num_vectors)
+    report_nonclose(inv_KFAC @ X, inv_KFAC_naive @ X, rtol=5e-2)
+    report_nonclose(inv_KFAC_tuple @ X, inv_KFAC @ X, rtol=5e-2)
```

Comment:
I was also confused by this, it might be the numerical instability of the inversion? I think I might have converted everything to `float64` to verify that it requires less `rtol`, but not 100% sure anymore. I think most of the tests comparing for exact equivalence would benefit from being performed in `float64`.

Reply:
Then, let's run this test in `float64` and hope we can get an `atol` which is a factor of 10 smaller.